lib/dockerregistry/storage_driver.go

```diff
@@ -66,6 +67,22 @@ func (e InvalidRequestError) Error() string {
 	return fmt.Sprintf("invalid request: %s", e.path)
 }
 
+// toDriverError wraps ErrNotExist and ErrBlobNotFound errors to
+// driver.PathNotFoundError.
+func toDriverError(err error, path string) error {
+	if err != nil {
```

Comment:
Is this check necessary? I sort of feel that we should discourage calling `toDriverError(nil, "blah")`